components/server/src/user/user-deletion-service.ts

```diff
@@ -220,6 +224,14 @@ export class UserDeletionService {
         await Promise.all(userProjects.map((project) => this.projectDb.markDeleted(project.id)));
     }
 
+    protected async deleteLinkedInProfile(userId: string) {
+        try {
+            await this.linkedInService.deleteLinkedInProfile(userId);
+        } catch (error) {
+            log.error({ userId: userId }, "Failed to delete user's LinkedIn profile.", error);
```

Comment:
The other methods in the deletion lifecycle throw their errors back. Is the catch here deliberate?

Reply:
Happy to discuss this. In the past, we've had cases where various small random things would fail during user deletion, which completely prevented users to delete their account. This was annoying to them, and created back-and-forth support overhead.

However, I'd also understand the case that we want this entire process to fail if any random small thing is not properly deleted. So, I'm happy to remove the try/catch if you feel that's better.

Reply:
Done ✅